### ISSUE TITLE
Not working when used via SSH, including WSL2 and remote machines

### ISSUE THREAD

**Issue by Matb85** [sveltejs/language-tools#1224]
**Describe the bug**
The extension is completely not functional when used via ssh (WSL 2 and remote machines) 

**To Reproduce**
Just open a Svelte project in WSL2 or in a remote machine
**When used with WSL2, the project is located within the WSL file system** (e.g. \\wsl.localhost\Ubuntu\home\path\to\the\project )

**Expected behavior**

**Screenshots**
![Screenshot (33)](https://user-images.githubusercontent.com/69219238/139469522-ccfbf8d6-316c-4632-a87e-0ccf587022a9.png)
![Screenshot (31)](https://user-images.githubusercontent.com/69219238/139469542-c60b9356-fe12-4937-9dd3-045ddc4ed653.png)

When I set "files.associations": {"*.svelte": "html"  }  inside my VSCode settings I do get syntax highlight of basic html/js/css elements but as you can see svelte syntax e.g. {#each } still does not work, nor does peeking

![Screenshot (35)](https://user-images.githubusercontent.com/69219238/139846941-8fc21646-a0ce-4c6f-9d64-0896212071c6.png)

Interestingly, when I open the folder in Windows everything looks and works fine:

![Screenshot (36)](https://user-images.githubusercontent.com/69219238/139848346-50a32fc7-9beb-47b5-a674-213aa604428f.png)

**System (please complete the following information):**
 - OS: Windows 11 Home 22000.258 Insider Build
 - IDE: VSCode
 - Plugin/Package: Svelte for VS Code v105.4.2

**Additional context**
I don't know if it's happened just to me, anyway, this situation makes coding almost impossible. I've also tried to open the project in VS Code via ssh on my local raspberry pi 4B and the situation is exactly the same. Is there a way to fix this? Am I missing something? Help appreciated!

**Comment by jasonlyu123** [comment]
I guess your extension doesn't install correctly. As the syntax highlight is handled by vscode. We only provide a syntax highlight config file. 

How did you connect to the WSL? Did you use the remote ssh extension or the remote WSL extension? The extension page looks kind of different with mine.

**Comment by Matb85** [comment]
I've already tried reinstalling the extension many times but to no avail

To connect to wsl2 I click the right bottom button or Ctrl+Shift+P and select: **Remote-WSL: Open Folder in WSL**, and for remote machine:  **Remote-SSH: Connect to Host**

**Comment by jasonlyu123** [comment]
![圖片](https://user-images.githubusercontent.com/36730922/140440888-48e29c6d-80d3-487d-90dc-e479b96ab6c0.png)
Mine looks like this in Remote-WSL.

![圖片](https://user-images.githubusercontent.com/36730922/140442576-c86ba7b2-a822-4cbf-bfb6-b4083fd70fd5.png)
In Remote-SSH

Notice that the extension is not grey out and it have a remote icon next to it.

**Comment by Matb85** [comment]
I see, for some reason, it didn't want to install on the remote machines. 

I reinstalled vs-code and it started working! Indeed, now there is a blue circle with an icon next to the Svelte logo just like in your screenshots. Also, I've noticed the extension has been upgraded to v105.5.0.
Though, kinda strange anyway 🤔
![image](https://user-images.githubusercontent.com/69219238/140702264-8b909e3b-918f-4ba3-965e-0e32127fa389.png)
![image](https://user-images.githubusercontent.com/69219238/140702944-38fab9c7-39ac-4329-9e88-453930680fbd.png)
